build.yaml

```diff
@@ -66,7 +77,8 @@ build:
       jdk_switcher use $JAVA_VERSION
   - type: maven
     version: 3.2.5
-    goals: verify --fail-never -P$TEST_GROUP
+    # for Nightly builds, only run the "long" tests against OpenJDK8
+    goals: verify --fail-never -P$(if [ "${NIGHTLY}" = "true" -a "${JAVA_VERSION}" != "openjdk8" ]; then echo "short"; else echo ${TEST_GROUP}; fi)
```

Comment:
Unfortunately this is failing on Jenkins, see: 

https://jenkins-drivers.build.dsinternal.org/view/java-driver/job/datastax.java-driver.java2518.commit/2/CASSANDRA_VERSION=3.11,JAVA_VERSION=openjdk11,OS_VERSION=ubuntu%2Fbionic64%2Fjava-driver/console

Reply:
I was afraid of that. I'm not sure I can easily adjust the `build.yaml` to switch between `long` and `short` tests within the nightly schedule currently. I'll keep trying a few things, but if it gets to the point where changing Job Creator is the only way, I'm not sure it's worth it at that point.

Reply:
I've reverted this and just left it a reduced matrix for nightly builds.